web/views.py

```diff
@@ -13,7 +14,8 @@ def about(request):
 	return render(request, 'pages/about.html',{})
 
 def members(request):
-	return render(request, 'pages/members.html',{})
+	members_list = UserProfile.objects.filter(active=True).order_by('activity_status')
```

Comment:
I was trying to group members by active_status only didn't think it'd be so difficult. It does not really matter, so changed it. Happy now? :wink: